pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
I think, this line can be removed. When I run `make codegen` in this branch, I don't get this line.

Reply:
@FxKu seem to this line depends of the golang version utilizing. Before adding this line, CI job has failed without it.

Reply:
Also saw these changes in several PRs

Reply:
Which CI job? Do have links to "other PRs"?

Reply:
https://github.com/zalando/postgres-operator/pull/1672/files#diff-ad834b7f56540d53b0c98651ad608508b7532e395e472f12f76d5970c9ae6da7

Reply:
I see, hmm:
```
diffing hack/../pkg against freshly generated codegen
diff -Naupr hack/../pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go hack/../_tmp/pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go
--- hack/../pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go	2021-12-02 07:21:30.048366700 +0000
+++ hack/../_tmp/pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go	2021-12-02 07:24:23.029626056 +0000
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
 /*
hack/../pkg is out of date. Please run 'make codegen'
make: *** [Makefile:103: test] Error 1
Error: Process completed with exit code 2.
```

Reply:
on my machine it's the opposite. When I add this line `verify-codegen` fails. Maybe should update to Go 1.17.

Reply:
Yes, indeed, but confusing a bit with CI is utilizing golang version 1.16.9

Reply:
Can you bring back the generated code line we talked about? Then this PR can be merged.

Reply:
Yes, sure. Done.